apps/saleor-app-checkout/constants.ts

```diff
@@ -14,12 +14,12 @@ export type DebugEnvVars = Record<DebugEnvVar, string | undefined>;
 
 // Need to use `var variable = process.env.VARIABLE;`, not `var env = process.env; var variable = env.VARIABLE;`
 // https://github.com/vercel/next.js/issues/19420
+export const envVars: EnvVars = {
+  apiUrl: process.env["NEXT_PUBLIC_SALEOR_API_URL"]!,
```

Comment:
We can't use process.env like that, due to how Next.js env variables processing works - I'm working on fix here: https://github.com/saleor/saleor-checkout/pull/174

```suggestion
  apiUrl: process.env.NEXT_PUBLIC_SALEOR_API_URL!,
```

Reply:
Yes, we can. We can't use variables to get anything front-end related from `process.env` but `process.env["NEXT_PUBLIC_SALEOR_API_URL"]` is fine

Reply:
but... it is frontend realted, this env variable it's used for creating urql client on frontend

Reply:
We talked in direct messages, it's all good!